pkg/indicator/klinestream.go

```diff
@@ -36,6 +36,10 @@ func (s *KLineStream) AddSubscriber(f func(k types.KLine)) {
 	}
 }
 
+func (s *KLineStream) AddBackLog(kLines []types.KLine) {
```

Comment:
How about rename this to BackfillKLines?

Reply:
Or maybe just Backfill?

Reply:
up to you